vendor/bat-native-ads/src/bat/ads/internal/ads/serving/targeting/models/contextual/text_embedding/text_embedding_aliases.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ADS_SERVING_TARGETING_MODELS_CONTEXTUAL_TEXT_EMBEDDING_TEXT_EMBEDDING_ALIASES_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ADS_SERVING_TARGETING_MODELS_CONTEXTUAL_TEXT_EMBEDDING_TEXT_EMBEDDING_ALIASES_H_
+
+#include <vector>
+
+#include "base/containers/circular_deque.h"
+#include "bat/ads/internal/ml/data/vector_data.h"
+
+namespace ads {
+namespace targeting {
+
+using TextEmbeddingList = base::circular_deque<ml::VectorData>;
```

Comment:
We have refactored the ads lib to not use `_aliases.h` files. Please some other `*List` examples

Reply:
Was able to remove text_embedding_aliases.h entirely